### ISSUE TITLE
"Error with getID3..." but file gets added anyway

### ISSUE THREAD

**Issue by jimbolaya** [Rello/audioplayer#357]
### Expected behavior
`sudo -uwww-data php /var/www/nextcloud/occ audioplayer:scan --debug username` should scan all filenames consistently when filename encoding is UTF-8.

### Actual behavior
When I run `sudo -uwww-data php /var/www/nextcloud/occ audioplayer:scan --debug username | tee scanlog` and review scanlog, I see a number of entries like this:

```
   /username/files/music/Beck/Guero (Deluxe Edition)/Beck - Still Missing (R
       Error with getID3. Does not seem to be a valid audio file
```
Where the actual filename is: 
`/username/music/Beck/Guero (Deluxe Edition)/Beck - Still Missing (Röyksopp Remix).mp3`

Other files with the offending character seem to have no issues.
`/username/files/music/Björk/Post/Björk - Army Of Me.mp3`

I've done extensive checks to make sure the filenames are encoded correctly in UTF-8. 

I'm not sure if this is a real problem or not as the files show up in the list of songs and are playable. The list of characters that trip up the script varies, but are all accented characters of some sort.

### Steps to reproduce the behavior
1. Run `sudo -uwww-data php /var/www/nextcloud/occ audioplayer:scan --debug username | tee scanlog`
2. Examine `scanlog` for filenames throwing `valid audio` messages

### Possible solution (optional)
Ignore, scan seems to work anyway.

### Server configuration
**Operating system:**
Debian Jessie (8.x) x86_64

**Web server:**
apache2    2.4.10-10+deb8u12

**Database:**
mysql-server-5.5   5.5.60-0+deb8u1

**Nextcloud/ownCloud version:** (see `/status.php`)
13.0.7

**Audio Player version:** (see `/settings/apps`)
2.4.1

**Are you using external storage? If yes, which one:** `local/smb/sftp/...`
Yes, local

### Client configuration
**Operating system:**
Linux

**Browser:**
Firefox

**Comment by Rello** [comment]
Hello,
sorry for the late reply.
I think what you are referring might be an issue thats coming up from time to time but the real issue is not clear.

Did I understand correctly, that you had an issue displayed - but when scanning again, it worked and all titles were added?

The "Error in getID3" is caused when the file could not be read.
This might be the case if

- the file is in fact defect. then it would show again as defect in the next scan
- the access to the file was interrupted. 

The last case was seen sometimes when php-smblibrary was not installed or php was failing die to another tmp cache issue. But in your case you don´t seem to use SMB.

If it worked at the end and you could not find anything usefull in the php- or the apache-logs, I think we need to keep it there for now.
I hope AP is still useful for you

let me know...

**Comment by jimbolaya** [comment]
The song gets added even if there is the error. 

Currently it's only happening with 5 out of my 3300-ish files.

It's possible that there's an issue with the ID3 encoding rather than the filename, but if that's the case I don't entirely understand why there's:

```
   /jklaas/files/music/Paul Simon/You're The One/Paul Simon - Se
       Error with getID3. Does not seem to be a valid audio file
```
when the filename is
`Paul Simon - Señorita With a Necklace Of Tears.mp3`

It happens with the same files every time I scan.

I did install "php-smblibrary", then I remembered that the external storage is local to the server, so while it may help in other cases, I can't imagine it would help here.

I don't see much in the way of php-* logs. is there some way for me to create or increase the logging?

I'm also happy just letting things lay for now, since it doesn't seem to be an issue. Thanks for following up.
